assets/in

```diff
@@ -72,7 +73,15 @@ for branch in $fetch; do
   git branch $branch FETCH_HEAD
 done
 
+return_ref=""
+if [ -n "$tag_filter" ]; then
+  return_ref=$(git tag -l --points-at HEAD "$ref")
+  [ -n "$return_ref" ] || return_ref=$(git tag -l --points-at HEAD | tail -n 1)
+fi
+[ -n "$return_ref" ] || return_ref=$(git rev-parse HEAD)
+echo $return_ref
```

Comment:
Sorry, you are right I missed that one. The resource behaviour was not affected thx to redirect stdout to 3 fd.